infrasec/aws/naming.md

```diff
@@ -77,3 +92,89 @@ e.g.
 
 - slack-pivotal-tracker-bot-test
 - rds-log-cleaner-webapp-prod
+
+### SNS topics
+
+The main purpose of the SNS topics are for notifications to teams. The naming convention reflects that, closely followed by the AWS account type (Gov or Com) and action.
+
+*${team}-${account_type}-${action}* - is the general form
```

Comment:
```suggestion
`${team}-${account_type}-${action}` - is the general form
```
Not blocking: subjectively, I find this a bit easier to read than italics. There are a few more instances later on also, if you agree. If not, it's no kind of big deal :)